org_chart.js

```diff
@@ -1,73 +1,73 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+function Person(name, email, skill, pronoun) {
+  this.email = email
+  this.managees = []
+  this.manager = {}
+  this.name = name
+  this.position = ''
+  this.project = ''
+  this.pronoun = pronoun
+  this.skills = [skill]
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
+Person.prototype.add_managee = function(person) {
+  let already_managing = false
+  this.managees.forEach(managee => {
+    if (managee === person) {
+      already_managing = true
+    }
+  })
+  if (!already_managing) {
+    this.managees.push(person)
   }
-]
-
-// assign each a manager
-let [alan, grace, donald] = new_hires
+}
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+Person.prototype.add_skill = function(skill) {
+  let has_skill = false
+  this.skills.forEach(existing_skill => {
+    if (existing_skill = skill) {
+      has_skill = true
+    }
+  })
+  if (!has_skill) {
+    this.skills.push(skill)
+  }
+}
```

Comment:
A more idiomatic way of doing this would be:
```js
Person.prototype.add_skill = function(skill) {
  this.skills.forEach( (current_skill) => {
    if (current_skill === skill) {
      return
    }
  }
  this.skills.push(skill)
}
```